src/test/pybind/test_ceph_argparse.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+# use the script to setup the nosetest environment as it should be.
+# Mainly due /usr/bin/env uses a parameter setup different from what
+# python expects
+
+[ ! -z $CEPH_ROOT ] || exit 1
+
+PYBIND=$CEPH_ROOT/src/pybind /usr/local/bin/nosetests  --stop $CEPH_ROOT/src/test/pybind/test_ceph_argparse.py
```

Comment:
@tchaikov 
Eh, that could be that case.
I first did the daemon case, and then copied the wrapper.
Even considered make it a generic one in taking arg0 and rewrite it to basename + .py.
What is your preference: ONE generic one, or that I check PYBIND usage.